lib/pliny/commands/creator.rb

```diff
@@ -24,13 +30,21 @@ def run!
       FileUtils.rm_rf("#{app_dir}/.git")
       setup_database_urls
       display 'Pliny app created. To start, run:'
-      display "cd #{app_dir} && bin/setup"
+      if opts[:docker]
+        display "cd #{app_dir} && docker-compose run web bin/setup"
```

Comment:
Does this actually work? When I run bin/setup under docker, it complains about dropping the database when there's a current connection. Also, bin/setup does other things like `bundle install` which shouldn't be necessary.